src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -98,6 +103,41 @@ public override async Task<ResourceElement> GetCapabilityStatementOnStartup(Canc
             return _listedCapabilityStatement;
         }
 
+        public async Task BackgroudLoop()
+        {
+            while (!_cancellationTokenSource.IsCancellationRequested)
+            {
+                for (int i = 0; i < _rebuildDelay; i++)
+                {
+                    await Task.Delay(TimeSpan.FromMinutes(1));
+
+                    if (!_cancellationTokenSource.IsCancellationRequested)
+                    {
```

Comment:
Should it be if cancellationRequested then return? Or I am missing something here

Reply:
Yes, thank you. Copy&paste error